pkg/virt-launcher/virtwrap/api/schema_test.go

```diff
@@ -359,6 +352,11 @@ var _ = Describe("Schema", func() {
 			table.Entry("for ppc64le", "ppc64le", exampleXMLppc64le),
 			table.Entry("for amd64", "amd64", exampleXML),
 		)
+		It("Unmarshal into struct for amd64 when device alias name has no user alias prefix", func() {
```

Comment:
Instead of this global test, can you create a scoped unittest which tests exactly the alias specific bits?

Like one test which starts with a non-user-alias:

```
alias := &Alias{name: "alias0", userDefined: false}
xmlBytes, err := xml.Marshal(alias)
Expect(err).ToNot(HaveOccured)
Expect(string(xmlBytes)).To(ContainSubstring("\"alias0\""))
newAlias = &Alias{}
Expect(xml.Unarshal(xmlBytes, newAlias)).To(Succeed())
Expect(newAlias.GetName()).To(Equal("alias0))
Expect(newAlias.name).To(Equal("alias0))
Expect(newAlias.userDefined).To(BeFalse())
```

and a second test:


```
alias := &Alias{name: "alias0", userDefined: true}
xmlBytes, err := xml.Marshal(alias)
Expect(err).ToNot(HaveOccured)
Expect(string(xmlBytes)).To(ContainSubstring("\"ua-alias0\""))
newAlias = &Alias{}
Expect(xml.Unarshal(xmlBytes, newAlias)).To(Succeed())
Expect(newAlias.GetName()).To(Equal("alias0))
Expect(newAlias.name).To(Equal("alias0))
Expect(newAlias.userDefined).To(BeTrue())
```

Reply:
This is good. Will put it.

Reply:
I added the two tests in. They show the marshal/unmarshal of Alias objects work as expected.